src/components/Books.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// import { useSelector } from 'react-redux';
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.